content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md

```diff
@@ -106,6 +107,11 @@ CSS styling. It should be added to a relevant site client library using the `emb
 It also provides a `core.wcm.components.image.v1.editor` editor client library category that includes
 JavaScript handling for dialog interaction. It is already included by its edit dialog.
 
+## SVG
+SVG MIME-types are supported, but have some specific handling. In contrast to lazy loading, which is still possible for SVG image types,
+alternative image sizes defined at the component policy dialog are ignored. In addition, SVG image types have a more limited set of
+editing options available in the AEM inline image editor.
```

Comment:
Can you already mention here which minimum AEM Version/Feature Pack is required for the limited editing options to be correctly applied?

Reply:
@kwin - we have plans to ensure the required product changes for both SVG and Image Map support are available on the currently supported AEM versions for the Core Components (6.3, 6.4). The exact feature or service pack is not yet known, but we will be updating our versioning documentation with a subsequent release of the core components.